server/routes/kubeconfig/kubeconfig_routes.py

```
@@ -0,0 +1,230 @@
+        return
+
+    vault_provider = _make_vault_key(org_id, filename)
+    try:
+        store_tokens_in_db(user_id, {"kubeconfig_yaml": content}, vault_provider, org_id=org_id)
+    except Exception:
+        logger.exception("[Kubeconfig] Vault store failed for %s", filename)
+        errors.append({"filename": filename, "error": "Failed to store credentials"})
+        return
+
+    with db_pool.get_user_connection() as conn:
+        with conn.cursor() as cur:
```

Ruff (0.15.15)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)